lib/rest/RequestHandler.js

```diff
@@ -275,7 +275,7 @@ class RequestHandler {
                             }
 
                             var stack = _stackHolder.stack;
-                            if(stack.startsWith("Error\n")) {
+                            if(typeof stack === "string" && stack.startsWith("Error\n")) {
```

Comment:
When would `stack` not be a string? As far as I can tell, there would never be a case where it's not a string.